### DISCUSSION TITLE
room assistant wont start

### DISCUSSION THREAD

**Discussion by kylevidrine** [mKeRix/room-assistant#1347] [General]
I keep getting this when trying to run room assistant. I have gone through the install docs and followed them to a T, however I am still getting this. The user mqtt is setup on my home assistant and mqtt is working with other intergrations.

● room-assistant.service - room-assistant service
     Loaded: loaded (/etc/systemd/system/room-assistant.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Mon 2023-01-23 07:01:09 CST; 3s ago
    Process: 4652 ExecStart=/opt/nodejs/bin/room-assistant (code=exited, status=217/USER)
   Main PID: 4652 (code=exited, status=217/USER)
        CPU: 16ms

**Comment by lorenzothurman** [comment]
Are you running on RPi?

**Comment by ghost** [comment]
Hi, I'm having this issue too and couldn't find any similar threads. I am running on a Pi 400 on Raspberry Pi OS Lite.
